File: java/dictionary/cars.java
import java.util.*;

public class CarDictionary {
    public static void main(String[] args) {
        // Create a HashMap to store car brands and their models
        Map<String, List<String>> cars = new HashMap<>();

        // Add entries to the map
        cars.put("Mazda", Arrays.asList("Miata", "RX7"));
        cars.put("Honda", Arrays.asList("S2000", "NSX"));
        cars.put("Toyota", Arrays.asList("Supra"));
        cars.put("Lexus", Arrays.asList("LFA", "LC500"));

        // Access and print specific entries
        System.out.println("Mazda: " + cars.get("Mazda"));
        System.out.println("Honda: " + cars.get("Honda"));
        System.out.println("Toyota: " + cars.get("Toyota"));
        System.out.println("Lexus: " + cars.get("Lexus"));

        // Loop through all keys and values
        System.out.println("All car brands and models:");
        for (Map.Entry<String, List<String>> entry : cars.entrySet()) {
            System.out.println(entry.getKey() + ": " + String.join(", ", entry.getValue()));
        }
    }
}

File: python/dictionary/cars.py
# Define the dictionary
cars = {
    "Mazda": ["Miata", "RX7"],
    "Honda": ["S2000", "NSX"],
    "Toyota": ["Supra"],
    "Lexus": ["LFA", "LC500"]
}

# Access and print specific entries
print("Mazda:", cars["Mazda"])
print("Honda:", cars["Honda"])
print("Toyota:", cars["Toyota"])
print("Lexus:", cars["Lexus"])

# Loop through all keys and values
print("All car brands and models:")
for brand, models in cars.items():
    print(f"{brand}: {', '.join(models)}")

